erpnext/accounts/party.py

```diff
@@ -149,12 +151,20 @@ def get_default_price_list(party):
 
 	return None
 
-def set_price_list(out, party, party_type, given_price_list):
+def set_price_list(out, party, party_type, given_price_list, pos=None):
 	# price list
 	price_list = get_permitted_documents('Price List')
 
 	if price_list:
 		price_list = price_list[0]
+	elif pos and party_type == 'Customer':
+		customer_price_list = frappe.get_value('Customer', party.name, 'default_price_list')
+
+		if customer_price_list:
+			price_list = customer_price_list
+		else:
+			pos_price_list = frappe.get_value('Pos Profile', pos, 'selling_price_list')
```

Comment:
`Pos Profile` -> `POS Profile`, otherwise this will always return `None`.

Maybe add a test for this.